python/dgl/data/graph_serialize.py

```diff
@@ -108,43 +118,56 @@ def save_graphs(filename, g_list, labels=None):
     _CAPI_DGLSaveGraphs(filename, gdata_list, label_dict)
 
 
-def load_graphs(filename, idx_list=None):
+def load_graphs(filename, idx_list=None, ignore_labels=True):
     """
     Load DGLGraphs from file
 
     Parameters
     ----------
     filename: str
-        filename to load DGLGraphs
+        filename to load graphs
     idx_list: list of int
         list of index of graph to be loaded. If not specified, will
         load all graphs from file
+    ignore_labels: bool
+        Whether to ignore the return of labels
 
     Returns
     ----------
-    graph_list: list of immutable DGLGraphs
-    labels: dict of labels stored in file (empty dict returned if no
+    
+    graph_list: list of DGLGraphs / DGLHeteroGraph
+    labels(Optional): dict of labels stored in file (empty dict returned if no
     label stored)
 
+    If the file is stored with labels (before DGL 0.4.4) and set ignore_labels=False,
+     it will return graph_list and labels
+    If the file isn't stored with labels, it will only return graph_list
+
+
     Examples
     ----------
     Following the example in save_graphs.
 
     >>> from dgl.data.utils import load_graphs
-    >>> glist, label_dict = load_graphs("./data.bin") # glist will be [g1, g2]
-    >>> glist, label_dict = load_graphs("./data.bin", [0]) # glist will be [g1]
+    >>> glist = load_graphs("./data.bin") # glist will be [g1, g2]
+    >>> glist = load_graphs("./data.bin", [0]) # glist will be [g1]
 
     """
     if idx_list is None:
         idx_list = []
     assert isinstance(idx_list, list)
-    metadata = _CAPI_DGLLoadGraphs(filename, idx_list, False)
+    metadata = _CAPI_LoadDGLGraphFiles(filename, idx_list, False)
     label_dict = {}
-    for k, v in metadata.labels.items():
-        label_dict[k] = F.zerocopy_from_dgl_ndarray(v.data)
-
-    return [gdata.get_graph() for gdata in metadata.graph_data], label_dict
-
+    if metadata.is_hetero:
+        g_list = [gdata.get_graph() for gdata in metadata.hetero_graph_data]
+    else:
+        for k, v in metadata.labels.items():
+            label_dict[k] = F.zerocopy_from_dgl_ndarray(v)
+        g_list = [gdata.get_graph() for gdata in metadata.graph_data], label_dict
+    if ignore_labels or len(label_dict) == 0:
+        return g_list

```

Comment:
Can we return g_list, None?

Reply:
In the previous dgl, it returns g_list, None. However from now on, we decided not to store labels with the graphs for simplicity. `save_graph(g_list)` no longer accepts `label_dicts`.
Therefore it doesn't make too much sense to return another value (None) for the future user?